src/libs/actions/Session/index.ts

```diff
@@ -205,6 +206,7 @@ function hasAuthToken(): boolean {
 function signOutAndRedirectToSignIn(shouldResetToHome?: boolean, shouldStashSession?: boolean, killHybridApp = true) {
     Log.info('Redirecting to Sign In because signOut() was called');
     hideContextMenu(false);
+    ShortcutManager.removeAllDynamicShortcuts();
```

Comment:
I think it would probably be better to put this in [PushNotification.deregister](https://github.com/Expensify/App/blob/f8188b0e9c8674e016c38196b3f23a3ed07d7cc4/src/libs/Notification/PushNotification/index.native.ts#L133-L142). Can we do a quick test to make sure it still works correctly from there?